docker/Dockerfile.ci-operator-buildroot

```diff
@@ -5,13 +5,14 @@
 
 FROM quay.io/fedora/fedora:36-x86_64
 
+# Disable color output to make log files more readable.
+ENV NO_COLOR=1
+
 # Update the system and install the necessary system packages.
 # Note that ci-operator requires git to be installed on the system.
 # https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-root-image
 RUN dnf update -y && \
-    dnf install git jq -y && \
-    git config --system user.name origin-release-container && \
-    git config --system user.email origin-release@redhat.com && \
+    dnf install -y git jq && \
```

Comment:
Removed git user name/email config because it's not really necessary for cloning the repo.